src/App.css

```diff
@@ -2,21 +2,31 @@
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
+.menu-container {
+  position: relative;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.menu {
+  background: #ffffff;
+  border-radius: 8px;
+  width: 300px;
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.3);
+  opacity: 0;
+  display: none;
+  text-align: center;
+  margin: 0;
+  transform: translateX(10%);
+  transition: opacity 0.4s ease, transform 0.4s ease, visibility 0.4s;
+}
+
+.menu.active {
+  opacity: 1;
+  display: inline-block;
+  transform: translateX(10%);
 }
 
-.App-link {
-  color: #09d3ac;
+@media only screen and (max-width: 700px) {
+  .menu.active {
+    transform: translateX(1%);
+  }
```

Comment:
I checked out the app since I was a little unsure of that the translates were doing here, and I think it's more effective to instead use flexbox containers that can align content without the need for media queries. You can remove any translate properties add something like this to `.menu-container`:

```
display: flex;
flex-direction: column;
align-items: center;
```

Reply:
thank you Luis for reviewing the PR, I didn't want to disturb the mentors. Thanks, again! I have deleted the transforms and added flexbox.